interface/forms/newpatient/common.php

```diff
@@ -86,16 +86,35 @@ function refreshIssue(issue, title) {
   s.options[s.options.length] = new Option(title, issue, true, true);
  }
 
+ <?php
```

Comment:
Note the above settings for option_id and title in list_options are just examples. Feel free to place your own stamp on this.